src/ui/qml/NetworkSetupWizard.qml

```diff
@@ -93,43 +93,110 @@ ApplicationWindow {
-                onClicked: {
-                    // Reset to defaults and proceed to bootstrap page
-                    configPage.reset()
-                    configPage.save()
+                Image {
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    source: "../icons/ricochet.svg"
+                    sourceSize.width: parent.width / 10
+                    sourceSize.height: 60
                 }
-            }
 
-            Rectangle {
-                height: 1
-                width: parent.width
-                color: palette.mid
+                Label {
+                    width: parent.width
+                    horizontalAlignment: Qt.AlignHCenter
+                    text: qsTr("Welcome to Ricochet. Before we start, we need to know about your connection to the Internet.")
+                    wrapMode: Text.Wrap
+                }
+                
+                Label {
+                    width: parent.width
+                    horizontalAlignment: Qt.AlignHCenter
+                    text: qsTr("Please try <b>Connect</b> first if you do not require bridges or special network settings.")
+                    wrapMode: Text.Wrap
+                }
             }
 
-            Label {
+            Row {
                 width: parent.width
-                text: qsTr("This computer's Internet connection is censored, filtered, or proxied. I need to configure network settings.")
-                wrapMode: Text.Wrap
-                horizontalAlignment: Qt.AlignHCenter
-            }
+                spacing: 10
+
+                Column {
+                    width: parent.width/2.5
+                    spacing: 10
+
+                    Image {
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        source: "../icons/connection_ok.svg"
+                        sourceSize.width: parent.width
+                        sourceSize.height: 50
+                    }
+
+                    Button {
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        text: qsTr("Connect")
+                        isDefault: true
+                        onClicked: {
+                            // Reset to defaults and proceed to bootstrap page
+                            configPage.reset()
+                            configPage.save()
+                        }
+                    }
+
+                    Label {
+                        width: parent.width
+                        text: qsTr("This computer can freely access the Internet")
+                        wrapMode: Text.Wrap
+                        horizontalAlignment: Qt.AlignHCenter
+                    }
+                }
 
-            Button {
-                anchors.horizontalCenter: parent.horizontalCenter
-                text: qsTr("Configure")
-                onClicked: window.openConfig()
+                Column {
```

Comment:
So this is just supposed to be a spacer?

    Item {
        width: parent.width / 5
        height: 1
    }

should perform the job OK (you just need to assign both a width and height to force it to take space).